### PR TITLE
applying operators and scopes

### PR DESCRIPTION
**Pull request by jaumejane90**
Operators and Scopes!

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,30 +1,25 @@
...
 $.ajax({
   url: options.url
 }).done(function(data) {
-  questions = data.questions
+  var responseCount = 0
+  var currentQuestion = 0
+  var responses = []
+  const {questions} = data
 
+  debugger
```
Comment (davidbarna) [inline src/index.js]: Did the tests really passed with this ??!?!

Comment (jaumejane90) [reply src/index.js]: yes! I'll remove it!

File: src/index.js [hunk 2]
```diff
@@ -180,21 +157,10 @@ $.ajax({
...
-        case 'inputs':
-          if (!!responses[i] && !!responses[i].join('')) {
-            responseCount++
-          }
-          break
-        default:
-          if (!!responses[i]) {
-            responseCount++
-          }
+    responseCount = 0
```
Comment (davidbarna) [inline src/index.js]: Ok missing test here :-)
Fair enough. We'll see it in class.

**Comment by davidbarna** [comment]
👍

**Pull request closed without merging**